.rubocop.yml

```diff
@@ -1,12 +1,34 @@
 AllCops:
```

Comment:
I think you'll address a lot of this in the ticket for adding Rubocop to OHQ, but here are some good resources:

- https://github.com/thoughtbot/guides/blob/main/ruby/.rubocop.yml
- https://github.com/rubocop/rubocop/blob/master/config/default.yml